samcli/cli/command.py

```diff
@@ -21,6 +21,7 @@
     "samcli.commands.deploy",
     "samcli.commands.logs",
     "samcli.commands.publish",
+    "samcli.commands.setup",
```

Comment:
can we call this `configure` to be consistent with awscli? Or `bootstrap` is also popular

Reply:
We cannot call this configure as that would conflict with a future command with a different purpose. bootstrap is an option